tensorbay/client/profile.py

```diff
@@ -20,6 +21,13 @@
 from .requests import Client
 
 _Callable = TypeVar("_Callable", bound=Callable[..., Response])
+_Column = [
```

Comment:
we are using these string all over this file, can we find a way to reuse them rather write the raw string everywhere?